resources/features/Sampling.feature

```diff
@@ -0,0 +1,106 @@
+Feature: Making a Sample
+    The sample creation method will ingest a frame and a set of stratification parameters to produce an output sample.
+    To do this, the pre-filtered dataset is sorted in ascending order by 'prn'. Each of the stratification stratas with
```

Comment:
Question - are you sorting the total input frame by PRN prior to actioning the individual strata?
It's not a problem if this allows the creation of each strata in PRN order themselves. It's a different approach to our current system which allocates to strata first and then sorts each strata by PRN.

Reply:
Yes